pkg/integrations/cmdapi/cmdapi.go

```diff
@@ -0,0 +1,13 @@
+package cmdapi
+
+var (
+	integrationsAllowedToRunStopFromCmdAPI = map[string]struct{}{
+		"nri-lsi-java": {},
```

Comment:
I'm wondering if it is possible instead of having it hardcoded maybe receive it from the command channel? 🤔 
So we can avoid to having to release every time a new integration is allowed.

Reply:
cmd API already validates this on their side, the requirement for agent level validation is that in case cmd channel gets compromised (like MITM hijack) it could be used to trigger arbitrary integration executions which is the point we want to avoid.

Reply:
Oh, I see. 👌